### ISSUE TITLE
Finishing off the Summer and Documentation Work

### ISSUE THREAD

**Issue by amichuda** [minimod-nutrition/minimod-opt#22]
@Saloni-Chopra 

This issue will serve as a log of the things that you've done and a list of things that will need to be finished before the end of the summer.

- [ ] Fix all issues in the pull request and make sure it can be merged
- [ ]  Fix typos
- [ ]  Include multi-line strings in correct place
- [ ]  make sure that descriptions are correct
- [ ]  Do the same in the `optimize.py` file for continuous optimization
- [ ]  Create pull request and merge into main branch
